site/docs/how-tos/grpc.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Call gRPC service
+description: How to call unary or bidirectional streaming gRPC service from a pipeline.
+---
+
+The [Stateless Transforms](../api/stateless-transforms.md#mapusingservice)
+section shows how to call and use services to transform items in the
+pipeline. One way to implement a remote service is using
+[gRPC](https://grpc.io/) - an open source universal RPC framework, which
+is available for many platforms and languages.
+
+The `hazelcast-jet-grpc` module makes it easy to perform calls to a
+gRPC service. Currently, two types of gRPC services are supported:
+
+- unary service
+- bidirectional streaming service.
+
+This how to guide shows how to use these services in your pipeline.
+
+## Module dependency
+
+Add a dependency on `hazelcast-jet-grpc` module to use the gRPC services
+
+```xml
+<dependency>
+    <groupId>com.hazelcast.jet</groupId>
+    <artifactId>hazelcast-jet-grpc</artifactId>
+    <version>4.1</version>
```

Comment:
should use the placeholder version tag

Reply:
this is still unresolved. I believe we also show dependencies as gradle, so let's follow the standard there.